rules/windows/defense_evasion_execution_lolbas_wuauclt.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/10/13"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/13"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Execution via abusing the Windows Update Client to load an arbitrary Image."
```

Comment:
```suggestion
description = "Identifies Execution via abusing the Windows Update Client to load an arbitrary DLL."
```

Reply:
I would also add a line about how this is used to evade defenses. I am sure all LOLbins have some similar explanatory verbiage.